src/System.Private.CoreLib/shared/System/Buffers/ConfigurableArrayPool.cs

```diff
@@ -100,13 +100,13 @@ public override T[] Rent(int minimumLength)
 
                 // The pool was exhausted for this buffer size.  Allocate a new buffer with a size corresponding
                 // to the appropriate bucket.
-                buffer = new T[_buckets[index]._bufferLength];
+                buffer = GC.AllocateUninitializedArray<T>(_buckets[index]._bufferLength);
```

Comment:
>AllocateUninitializedArray [](start = 28, length = 26)

I may just be paranoid here but could this be a breaking change if folks were expecting the buffers in the pool to be cleared? do we need a config to switch to allocating with the uninit API?

Reply:
> do we need a config to switch to allocating with the uninit API?

No. We are not maintaining bug-for-bug compatibility in CoreCLR. If somebody was depending on the buffer being zero initialized before and the app breaks on .NET Core 3.0, it is their bug and they need to fix it before upgrading to .NET Core 3.0.